mlflow/transformers/__init__.py

```diff
@@ -2761,6 +2802,32 @@ def is_uri(s):
                 error_code=BAD_REQUEST,
             )
 
+    def _wrap_input_in_prompt_template(self, input_data):
```

Comment:
```suggestion
    def _format_prompt_template(self, input_data):
```

can we rename this to `_format_prompt_template`?